web-services/client/src/main/java/datawave/webservice/model/ModelList.java

```diff
@@ -13,8 +15,23 @@
 @XmlRootElement(name = "ModelList")
 public class ModelList extends BaseResponse implements Serializable, HtmlProvider {
     
+    private String jqueryUri;
+    private String dataTablesUri;
+    private String defaultModelTableName;
```

Comment:
This should be called `modelTableName` not `defaultModelTableName` since it is the selected model table name requested by the user (which could be the default, but isn't guaranteed to be).

Reply:
Yes.  Agreed.